mmdet/datasets/api_wrappers/panoptic_evaluation.py

```diff
@@ -195,7 +198,11 @@ def pq_compute_multi_core(matched_annotations_list,
         file_client_args = dict(backend='disk')
         file_client = mmcv.FileClient(**file_client_args)
 
-    cpu_num = multiprocessing.cpu_count()
+    if cpu_num == -1:
```

Comment:
1. cpu_num -> nproc, indicating number of multi-processing. By default, it can be 32.
2. we should set cpu_num = min(multiprocessing.cpu_count(), nproc), so that it can the number of process is suitable for the current number of CPU and avoid exceeding the memory limitation.